checkpoint/mysql/mysql.go

```diff
@@ -0,0 +1,158 @@
+package mysql
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"sync"
+	"time" // this is the mysql package so it makes sense to be here
```

Comment:
can we move this comment down, not sure it make sense next to "time"

Reply:
We can just drop it altogether. That was from the copy pasta from postgres implementation.